### PR TITLE
[TS] LPS-190886

### PR DESCRIPTION
**Pull request by yuhai**
@dantewang, the previous pr is https://github.com/liferay-core-infra/liferay-portal/pull/5568.

cc @marianoalvarosaiz

### CODE REVIEW

**Comment by Kvale1733** [comment]
@yuhai No regressions. Testing passed following ticket steps.

File: modules/apps/portal-cache/portal-cache-impl/src/main/java/com/liferay/portal/cache/internal/dao/orm/EntityCacheImpl.java [hunk 1]
```diff
@@ -306,52 +302,51 @@ private boolean _isLocalCacheEnabled() {
...
 
-			_notify(className, baseModel, removePortalCache);
+			_notify(className, baseModel, clearCache, removeCache);
 		}
 	}
 
 	private void _notify(
-		String className, BaseModel<?> baseModel, Boolean removePortalCache) {
+		String className, BaseModel<?> baseModel, boolean clearCache,
+		boolean removeCache) {
```
Comment (dantewang) [inline modules/apps/portal-cache/portal-cache-impl/src/main/java/com/liferay/portal/cache/internal/dao/orm/EntityCacheImpl.java:316]: @yuhai @marianoalvarosaiz Here we use 2 boolean parameters for the same if-else clause. Semantically, these 2 boolean parameters can not be both `true`, but can be one `true` + one `false`, or both `false`. However, we can't restrict it based on the current design.

If this change is not necessary for the fix, we can keep the original one; if you insist to make the change, I'd recommend using an `enum` or a selection of constants.

**Review by dantewang — CHANGES_REQUESTED** [review]

File: modules/apps/portal-cache/portal-cache-impl/src/main/java/com/liferay/portal/cache/internal/dao/orm/EntityCacheImpl.java (hunk 1 above)
Comment (yuhai) [reply modules/apps/portal-cache/portal-cache-impl/src/main/java/com/liferay/portal/cache/internal/dao/orm/EntityCacheImpl.java]: @dantewang, I removed the last SF commit. Will trigger ci test again

cc @marianoalvarosaiz

**Comment by yuhai** [comment]
ci:test:sf

**Comment by yuhai** [comment]
ci:test:relevant

**Comment by Kvale1733** [comment]
ci:test:relevant

**Comment by Kvale1733** [comment]
@yuhai No regressions found. Testing passed following the ticket steps.

**Review by dantewang — APPROVED** [review]

**Comment by shuyangzhou** [comment]
ci:forward:force

**Pull request closed without merging**